examples/tests/valid/for_loop.w

```diff
@@ -16,4 +16,16 @@ for word in words {
       break;
     }
   }
+}
+
+for x in 0..5 {
+  assert(x >= 0);
+  assert(x < 5);
+  print("increment => 0 <= ${x} < 5");
+}
+
+for x in 10..5 {
```

Comment:
Can you add tests for using variables and expressions instead of literal values:

E.g.

```js
for x in y..(z*2) {
  // ...
}
```

Reply:
@eladb, the `for n in 0..3` statement gives `n` the values 0, 1, and 2, similar to how it works in Rust.

However, in Rust, if we use the syntax `for n in 0..=3`, it gives `n` the values 0, 1, 2, and 3.

Do you think it would be a good idea to make the Wing for loop behave the same way as Rust in this case?